### DISCUSSION TITLE
Running RetroArch on rpi-5

### DISCUSSION THREAD

**Discussion by adam-neeley** [nix-community/raspberry-pi-nix#52] [General]
I'm trying to set up a retro gaming system on a Raspberry Pi 5 using retroarch. I got the example working, but enabling xserver, lightdm, and retroarch just gives a black screen with a blinking cursor. Is there additional configuration required to get xserver working? Thanks.

**Comment by tstat** [comment]
Hmm, I have set up a pi 5 using retroarch and this project before. I'm out of town right now but will post that config when I'm back home next week.

**Reply by adam-neeley** [reply]
Thanks :sunglasses:

**Reply by adam-neeley** [reply]
Importing the appropriate nixos-hardware module seems to have done the trick, although I'm still curious what your config looks like.
